src/core/session.cpp

```diff
@@ -361,16 +361,22 @@ bool session::get_last_insert_id(std::string const & sequence, long & value)
 
 details::once_temp_type session::get_table_names()
 {
+    ensureConnected(backEnd_);
```

Comment:
OK, but let's do it after I merge (if all goes well...) the real change (coming soon).